include/qfusion.hpp

```diff
@@ -223,10 +226,11 @@ class QFusion : public QInterface {
         }
     }
 
-    void FlushVec(const std::vector<bitLenInt> bitList)
+    void FlushSet(std::set<bitLenInt> bitList)
```

Comment:
Why does the order that these are processed in matter? (regarding vector -> set change).

Reply:
See my comment above: it's the uniqueness of elements that matters more, but we also iterate over a `find()` here in the `FlushBit()`.